deploy/operator-group.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: my-group
+  namespace: default
```

Comment:
this has to be tektoncd-pipelines or tekton-pipelines so that the operator is in the same ns as pipeline

Reply:
`tekton-pipelines`